packages/material-ui-system/src/index.d.ts

```diff
@@ -288,6 +288,23 @@ export interface CSSOthersObjectForCSSObject {
   [propertiesName: string]: CSSInterpolation;
 }
 
+export type SystemProps = PropsFor<
```

Comment:
Extracted so that the typings can be used in both the `Box`, `Typogrpahy` and the other CSS components we'll introduce.